File: pidev/src/Entite/Articles.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entite;

import java.util.logging.Logger;

/**
 *
 * @author khaled22
 */
public class Articles {
    private int id ;          private String titre ;
    private String sub_t ;    private String descript ;
    private String image ;    private int id_user ;

    public Articles(String titre, String sub_t, String descript, String image, int id_user) {
        this.titre = titre;
        this.sub_t = sub_t;
        this.descript = descript;
        this.image = image;
        this.id_user = id_user;
    }

    public Articles(int id) {
        this.id = id;
    }
    private static final Logger LOG = Logger.getLogger(Articles.class.getName());

    public int getId() {
        return id;
    }                   public void setId(int id) {
        this.id = id;
    }

    public String getTitre() {
        return titre;
    }             public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getSub_t() {
        return sub_t;
    }             public void setSub_t(String sub_t) {
        this.sub_t = sub_t;
    }

    public String getDescript() {
        return descript;
    }          public void setDescript(String descript) {
        this.descript = descript;
    }

    public String getImage() {
        return image;
    }             public void setImage(String image) {
        this.image = image;
    }

    public int getId_user() {
        return id_user;
    }              public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    @Override
    public int hashCode() {
        int hash = 3;
        hash = 97 * hash + this.id;
        hash = 97 * hash + this.id_user;
        return hash;
    }                   @Override 
                                                       public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Articles other = (Articles) obj;
        if (this.id != other.id) {
            return false;
        }
        if (this.id_user != other.id_user) {
            return false;
        }
        return true;
    }

                                       @Override
                             public String toString() {
        return "Articles{" + "id=" + id + ", titre=" + titre + ", sub_t=" + sub_t + ", descript=" + descript + ", image=" + image + ", id_user=" + id_user + '}';
    }
    
    
    
    
    
}

File: pidev1/src/AppBundle/Entity/Articles.php
<?php

namespace AppBundle\Entity;

/**
 * Articles
 */
class Articles
{
    /**
     * @var integer
     */
    private $titre;

    /**
     * @var integer
     */
    private $subT;

    /**
     * @var string
     */
    private $descript;

    /**
     * @var string
     */
    private $image;

    /**
     * @var integer
     */
    private $idUser;

    /**
     * @var integer
     */
    private $idArticle;


    /**
     * Set titre
     *
     * @param integer $titre
     *
     * @return Articles
     */
    public function setTitre($titre)
    {
        $this->titre = $titre;

        return $this;
    }

    /**
     * Get titre
     *
     * @return integer
     */
    public function getTitre()
    {
        return $this->titre;
    }

    /**
     * Set subT
     *
     * @param integer $subT
     *
     * @return Articles
     */
    public function setSubT($subT)
    {
        $this->subT = $subT;

        return $this;
    }

    /**
     * Get subT
     *
     * @return integer
     */
    public function getSubT()
    {
        return $this->subT;
    }

    /**
     * Set descript
     *
     * @param string $descript
     *
     * @return Articles
     */
    public function setDescript($descript)
    {
        $this->descript = $descript;

        return $this;
    }

    /**
     * Get descript
     *
     * @return string
     */
    public function getDescript()
    {
        return $this->descript;
    }

    /**
     * Set image
     *
     * @param string $image
     *
     * @return Articles
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * Set idUser
     *
     * @param integer $idUser
     *
     * @return Articles
     */
    public function setIdUser($idUser)
    {
        $this->idUser = $idUser;

        return $this;
    }

    /**
     * Get idUser
     *
     * @return integer
     */
    public function getIdUser()
    {
        return $this->idUser;
    }

    /**
     * Get idArticle
     *
     * @return integer
     */
    public function getIdArticle()
    {
        return $this->idArticle;
    }
}

